maintenance/wikia/cronjobs/AutoFollow.php

```diff
@@ -46,11 +57,31 @@
 		continue;
 	}
 
+	foreach ( $textsToWatch[$contentLanguage] as $text ) {
+		$titlesToWatch[] = Title::newFromText( $text );
+	}
+
+	$status = false;
 	foreach ( $titlesToWatch as $title ) {
-		echo "Watching {$title->getText()} as {$user->getName()}...\n";
-		WatchAction::doWatch( $title, $user );
+		$logParams = [
+			'user_id' => $row['user_id'],
+			'user_name' => $user->getName(),
+			'user_lang' => $userLanguage,
+			'title' => $title->getText(),
```

Comment:
Though we know this is in the blog namespace so not including the namespace isn't that big of a problem, I think logging `$title->getPrefixedText()` would be better as it is the full page name being watched.